### ISSUE TITLE
com.wsscode.log dependency has CLJS-incompatible var in it (not present in wilkerlucio/log)

### ISSUE THREAD

**Issue by tekacs** [wilkerlucio/pathom3#118]
This issue is related in shape to https://github.com/wilkerlucio/pathom3/pull/117

When I import pathom client-side, I see the following warning from `com.wsscode.log`:

```
------ WARNING #1 - :undeclared-var --------------------------------------------
 Resource: com/wsscode/log.cljc:57:18
 Use of undeclared Var com.wsscode.log/*file*
--------------------------------------------------------------------------------
```

The reason I'm posting this to pathom3 rather than [wilkerlucio/log](https://github.com/wilkerlucio/log) is because the version of `log.cljc` that exists in the shipped JAR of Pathom 2021.12.10-alpha... seems to differ from the version seen in pathom3's deps.edn.

Pathom3's deps.edn names a particular Git SHA, as of Pathom 2021.12.10-alpha:
https://github.com/wilkerlucio/pathom3/blob/6c33ce8158bdcc95aa5ea26077c208bf12fbc90a/deps.edn#L8-L9

The variant of wilkerlucio/log on Github at that SHA can be found here: https://github.com/wilkerlucio/log/blob/325675d8d2871e74e789e836713ea769139e9c61/src/main/com/wsscode/log.cljc

The variant of wilkerlucio/log found in the JAR of Pathom 2021.12.10-alpha looks like this:
https://gist.github.com/tekacs/0f64dbbffaee49f936137db2b2df6c45

Maybe I'm missing something very obvious, but I couldn't find a copy of this latter `log.cljc` anywhere in the wilkerlucio/log repo, nor online -- and certainly not at the SHA named by pathom3's deps.edn.

Again, I might be missing an obvious explanation, but is it perhaps possible that a `:local/root` version of `com.wsscode.log` made its way into the JAR for Pathom 2021.12.10-alpha? It looks like Github Actions is used to test, but not to release Pathom. Could a local copy have made it in perhaps?

**Comment by wilkerlucio** [comment]
Yes, you right, I just confirmed that for some reason the local version of my `log` was in mismatch with the remote one. Thanks!

I just fixed that, releasing a new version soon.

**Comment by wilkerlucio** [comment]
Released in `2022.01.09-alpha`
